portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,176 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+* {
+  font-family: Helvetica;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Header styling */
```

Comment:
Please order this file declarations alphabetically.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
done